.github/workflows/build.yml

```diff
@@ -14,15 +14,15 @@ jobs:
         include:
           - platform: ubuntu-20.04
             lsb_release: focal
-            otp-version: 23.1-1
+            otp-version: 23.2.3-1
             kafka-version: 2.4
           - platform: ubuntu-20.04
             lsb_release: focal
             otp-version: 22.3.4.8-1
-            kafka-version: 2.2
+            kafka-version: 1.1
           - platform: ubuntu-20.04
             lsb_release: focal
-            otp-version: 24.0.2-1
```

Comment:
oh, ci provided rebar3 is built on otp 22.
we should test with otp24 for both brod and kafka_protocol.

Reply:
Please, review it: dd3a0c446da73dbb14b56f2182e24d7accbee1e7